viv-wallet-app/src/views/Consultants.vue

```diff
@@ -24,7 +24,16 @@
                         >Ajouter</router-link
                     >
                 </div>
-                <consultant-list v-bind:consultants="consultants" v-bind:expertise="selectedExpertiseId" />
+                <consultant-list v-bind:consultants="consultants" v-bind:expertise="selectedExpertiseId">
+                    <template v-slot="{ consultantId }">
```

Comment:
interesting

Reply:
I finally found the right way to do component compositions in Vuejs.